plugins/kubernetes/app/models/kubernetes/build_job_executor.rb

```diff
@@ -91,6 +94,23 @@ def fill_job_details(k8s_job, build, project, tag:, push: false, tag_as_latest:
       k8s_job[:spec][:template][:spec][:containers][0].update(container_params)
     end
 
+    # So far we don't have ruby API for clair scanner and using shell script to wrap hyperclair.
+    # Hyperclair will pull the image from registry and run scan against Clair scanner
+    def scan_with_clair(project, tag)
+      require 'mixlib/shellout'
+      cmd = Mixlib::ShellOut.new("/usr/local/bin/clair_scan.sh", project, tag, @registry[:serveraddress])
+      cmd.live_stream = STDOUT
+      if File.executable?("/usr/local/bin/clair_scan.sh")
+        cmd.run_command
+        if cmd.error?
+          @output.puts "### Clair scanner found vulnarabilities, stopping build"
+          raise cmd.error!
```

Comment:
this looks strange ... what does it do ?
... should it not show what went wrong in the `@output` ?